Source: Timothy Redaelli <timothy@redaelli.eu>
Upstream: http://code.google.com/p/libarchive/issues/detail?id=21
Reason: test_option_t reference stdout is wrong.
	Tested with Linux and FreeBSD 6.4 with bsdcpio and GNU cpio

--- libarchive-2.6.992a/cpio/test/test_option_tnv.stdout.uu.orig	2009-04-16 12:02:29.835547984 +0200
+++ libarchive-2.6.992a/cpio/test/test_option_tnv.stdout.uu	2009-04-16 12:15:29.964560033 +0200
@@ -1,5 +1,5 @@
 begin 644 test_option_tnv.stdout
-M+7)W+7(M+7(M+2`@(#$@,3`P,"`@("`@,3`P,"`@("`@("`@("`@(#`@1&5C
-/(#,Q("`Q.38Y(&9I;&4*
+M+7)W+7(M+7(M+2`@(#$@,3`P,"`@("`@,3`P,"`@("`@("`@("`@(#`@2F%N
+/("`Q("`Q.3<P(&9I;&4*
 `
 end
